tvm_utility/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(tvm_utility)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+find_package(autoware_build_flags REQUIRED)
+find_package(OpenCV REQUIRED)
+
+# set dependencies for the tests
+set(CATKIN_TEST_DEPENDS pcl_ros autoware_msgs)
+
+find_package(
+  catkin REQUIRED
+  COMPONENTS tvm_vendor ${CATKIN_TEST_DEPENDS}
+)
+
+# declare dependency packages to catkin
+catkin_package(
+  INCLUDE_DIRS include
+  CATKIN_DEPENDS ${CATKIN_TEST_DEPENDS}
+)
+
+include_directories(
+  include
+  ${catkin_INCLUDE_DIRS}
+  ${tvm_vendor_INCLUDE_DIRS}/tvm_vendor
+)
+
+if(CATKIN_ENABLE_TESTING)
+  find_package(rostest REQUIRED)
+
```

Comment:
Please add `roslint_add_test()`

Reply:
Note: After this, you will have a lot of linting errors to fix when you run `catkin_run_tests`. Please address these.

Reply:
Done